README.md

```diff
@@ -25,13 +25,49 @@ And 4.2.14+amd64 would only apply to the amd64 release image.
 ### Add Releases To Channels
 
 Edit the appropriate file in `channels/`.
-For example, to add a release to stable-4.2 you would edit `channels/stable-4.2.yaml`.
-Channel semantics are documented [here][channel-semantics].
+For example, to add a release to candidate-4.2 you would edit [`channels/candidate-4.2.yaml`](channels/candidate-4.2.yaml).
 
 The file contains a list of versions.
 Please keep the versions in order.
 And LEAVE COMMENTS if you skip a version.
 
+#### Feeder Channels
+
+Channel semantics, as documented [here][channel-semantics], show nodes and edges being promoted to successive channels as they prove their stability.
+For example, a 4.2.z release will appear in `candidate-4.2` first.
+Upon proving itself sufficiently stable in the candidate channel, it will be promoted into `fast-4.2`.
+Some time after landing in `fast-4.2`, it will appear in `stable-4.2`.
```

Comment:
Text suggestion:
Upon proving itself sufficiently stable in the candidate channel, it will be promoted into `fast-4.2`.
After the release stays in fast channel, it will appear in `stable-4.2`.
*Note*:  We are doing the delayed publish of builds in to stable till we have phased rollout in stable channel. Once we have the phased rollout capability we intend to publish a release to both fast and candidate release at one go.

Reply:
Not quite your wording, but added a note about phased release rollouts in 9b9497b7b448.  File another comment if you have issues with the new wording.